presto-cassandra/src/test/java/com/facebook/presto/cassandra/TestCassandraTokenSplitManager.java

```diff
@@ -0,0 +1,46 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.cassandra;
+
+import com.datastax.driver.core.Cluster;
+import com.facebook.presto.cassandra.CassandraTokenSplitManager.TokenSplit;
+import com.google.common.collect.ImmutableList;
+import io.airlift.json.JsonCodec;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static io.airlift.testing.Assertions.assertGreaterThan;
+
+public class TestCassandraTokenSplitManager
+{
+    private static final int DOES_NOT_MATTER = 1;
+
+    @Test(enabled = false)
+    public void testCassandraTokenSplitManager()
+            throws Exception
+    {
+        CassandraSession cassandraSession = new CassandraSession(
+                "test",
+                ImmutableList.of("cassandra"),
+                Cluster.builder(),
+                DOES_NOT_MATTER,
+                DOES_NOT_MATTER,
+                JsonCodec.listJsonCodec(ExtraColumnMetadata.class),
+                DOES_NOT_MATTER);
+        CassandraTokenSplitManager tokenSplitManager = new CassandraTokenSplitManager(cassandraSession, 10);
+        List<TokenSplit> splits = tokenSplitManager.getSplits("test", "lineitem");
```

Comment:
If you intend to leave this commit, note the prereqs of test setup (e.g. a Cassandra cluster with a lineitem table in keyspace test)